Errors in integration tests

Executing `make check TEST=integration` I'm receiving following output:
```
make test
make[1]: Entering directory '/home/kzab/work/src/github.com/intelsdi-x/snap-plugin-publisher-cassandra'
bash -c "./scripts/test.sh integration"
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/gcc.dg/cpp: read /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/gcc.dg/cpp/dir-only-6.c: syntax error
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/gcc.dg/cpp/trad: read /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/gcc.dg/cpp/trad/comment.c: syntax error
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test: found packages main (235.go) and _ (blank1.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/bench/shootout: C source files not allowed when not using cgo or SWIG: binary-tree.c chameneosredux.c fannkuch.c fasta.c k-nucleotide.c mandelbrot.c meteor-contest.c nbody.c pidigits.c regex-dna.c reverse-complement.c spectral-norm.c threadring.c
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/ddd2.dir: found packages ddd (ddd2.go) and main (ddd3.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/ddd2.dir
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/dwarf: found packages ignored (dwarf.go) and main (linedirectives.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/dwarf
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs: 
Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug050.go:7:1: expected 'package', found 'func'
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug083.dir: found packages bug0 (bug0.go) and bug1 (bug1.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug083.dir
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug088.dir: found packages bug0 (bug0.go) and main (bug1.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug088.dir
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug106.dir: found packages bug0 (bug0.go) and bug1 (bug1.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug106.dir
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug133.dir: found packages bug0 (bug0.go) and bug1 (bug1.go) in /home/kzab/Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/bug133.dir
can't load package: package ./Downloads/gcc-6.1.0/gcc/testsuite/go.test/test/fixedbugs/b
...

```

--- a/scripts/test.sh
+++ b/scripts/test.sh
@@ -103,6 +103,5 @@ elif [[ $TEST_SUITE == "integration" ]]; then
 	go get github.com/smartystreets/goconvey/convey
 	go get github.com/smartystreets/assertions
 	
-	cd $SNAP_PLUGIN_SOURCE
 	SNAP_CASSANDRA_HOST=127.0.0.1 go test -v --tags=integration ./...
 fi

